packages/components/tabs/src/TabsList.tsx

```diff
@@ -134,19 +133,7 @@ export const TabsList = forwardRef<HTMLDivElement, TabsListProps>(
     return (
       <div className={wrapperStyles({ className })} ref={wrapperRef}>
         {arrows.prev !== 'hidden' && (
-          <Button
-            shape="square"
-            intent="surface"
-            size="sm"
-            className={navigationArrowStyles()}
-            onClick={handlePrevClick}
-            disabled={arrows.prev === 'disabled'}
-            aria-label="Scroll left"
-          >
-            <Icon>
-              <ArrowVerticalLeft />
-            </Icon>
-          </Button>
+          <Components.Prev handleClick={handlePrevClick} disabled={arrows.prev === 'disabled'} />
```

Comment:
I personnally don't like this way to write code... Also if the only purpose of this change is to comply to ESlint rule, maybe we should change/disable it, wdyt? :fist_raised:

Reply:
We can disable the rule for this file, but imho, we should keep this rule. Getting a warning when a function exceeds 100 lines serves as a good guardrail, indicating when we should consider splitting a function into smaller pieces.

Reply:
I wish we had a way to differentiate normal functions from component functions, which contain markup and can quickly grow in number of lines

Reply:
Found a middle ground and increased the threshold for `tsx` files to 150 lines. 👍

Reply:
I get the point, but then maybe we could simply put those buttons components code in separate file(s), wdyt? They shouldn't be exported at package level but internally it seems it could be a good middle ground?